rca/project_styleguide/templates/patterns/pages/programmes/programme_index.yaml

```diff
@@ -1,26 +1,27 @@
 context:
   page:
-    title: Test listing page
-    introduction: Test intro for this listing page
+    title: Our Programmes
+    introduction: <p>Experience the world&apos;s <a href="https://www.example.com/">number one</a> university for art and design.</p>
+    search_placeholder_text: What are you interested in?
   subpages:
     object_list:
       exists: true
       specific:
         - title: Test sub page
-          listing_image: fake
+          hero_image: fake
```

Comment:
If we _do_ work in the listing_image it's worth changing one of these to use it so we can see it working in the pattern guide.